apps/ejabberd/src/mod_mam_muc.erl

```diff
@@ -300,6 +304,19 @@ is_action_allowed(Action, From, To = #jid{lserver = Host}) ->
         default -> is_action_allowed_by_default(Action, From, To)
     end.
 
+-spec packet_to_search_body(Host :: ejabberd:server(), Packet :: xmlel()) ->
+    string().
+packet_to_search_body(Host, Packet) ->
```

Comment:
Do we need to have this new function duplicated between `mod_mam.erl` and `mod_mam_muc.erl`? It sounds like a good fit for `mod_mam_utils`.